docs/general/polkadot-v1.md

```diff
@@ -86,70 +81,73 @@ Decentralization is a crucial aspect of blockchain networks, but there is a trad
 
 Ideally, a network should be decentralized "enough" to make it practically impossible for someone to
 exert manipulative or malicious influence on the network. So, **decentralization is a tool while the
-goal is resilience**, which is achieved by additionally providing on-chain treasury and governance
+goal is resilience**, which is achieved by additionally providing on-chain treasury and open governance
 mechanism allowing continuous incentives for the network's participants without relying on
 intermediaries or centralized entities.
 
-Currently, Polkadot 1.0 achieves resilience through the following strategies:
+Polkadot 1.0 achieved resilience through the following strategies:
 
-- Nominated Proof of Staking (NPoS) where the stake per validator is maximized and evenly
-  distributed across validators.
-- The [Decentralized Nodes Program](https://nodes.web3.foundation/) aims to incentivize new operators to
+- **Nominated Proof-of-Stake (NPoS)** where the stake per validator is maximized and evenly
+  distributed across validators, and participation in native staking is decentralized and democratized as much as possible.
+- [**ELVES protocol**](../learn/learn-parachains-protocol.md) allows for the shared security system and the "pessimistic" rollup system, where everything coming through Polkadot's virtual cores passes rigorous checking.
```

Comment:
Tiny nit, but maybe we can call it cynical, as per this ;) https://x.com/baderyo_o/status/1937901346780479548

Reply:
I am a bit reluctant to use the term "cynical" that hasn't been widely adopted yet. Imo it is better to stick with Polkadot rollup for now